libkbfs/conflict_resolver.go

```diff
@@ -1436,13 +1441,19 @@ func (cr *ConflictResolver) fixRenameConflicts(ctx context.Context,
 	}
 
 	mergedNodeCache := newNodeCacheStandard(cr.fbo.folderBranch)
-	mergedNodeCache.GetOrCreate(mergedChains.mostRecentMD.data.Dir.BlockPointer,
-		string(mergedChains.mostRecentMD.GetTlfHandle().GetCanonicalName()), nil)
+	rootNode, _ := mergedNodeCache.GetOrCreate(
+		mergedChains.mostRecentMD.data.Dir.BlockPointer,
+		string(mergedChains.mostRecentMD.GetTlfHandle().GetCanonicalName()),
+		nil)
 	nodeMap, err := cr.fbo.blocks.SearchForNodes(
 		ctx, mergedNodeCache, ptrs, newPtrs, mergedChains.mostRecentMD)
 	if err != nil {
 		return nil, err
 	}
+	// Print out the root node here, to guarantee it isn't GC'd before
+	// SearchForNodes completes.
+	cr.log.CDebugf(ctx, "Found %d nodes for %d ptrs, root node %p",
```

Comment:
Looks like go 1.7 will have what you need: https://tip.golang.org/pkg/runtime/#KeepAlive . Maybe add a TODO to use it once we move to go 1.7.